modules/installation-three-node-cluster.adoc

```diff
@@ -31,7 +31,12 @@ endif::[]
 [id="installation-three-node-cluster_{context}"]
 = Configuring a three-node cluster
 
+ifndef::ibm-z,ibm-z-kvm[]
 You can optionally deploy zero compute machines in a bare metal cluster that consists of three control plane machines only. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for testing, development, and production.
+endif::ibm-z,ibm-z-kvm[]
+ifdef::ibm-z,ibm-z-kvm[]
+You can optionally deploy zero compute machines in a minimal three node cluster that consists of three control plane machines only. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for testing, development, and production.
```

Comment:
Change "You can optionally deploy" to "Optionally, you can deploy."

Reply:
Also changing it in line 35 to be consistent.